src/helpers/assets.ts

```diff
@@ -86,34 +36,23 @@ export const buildCoinsList = (
   nativeCurrency: any,
   isCoinListEdited: any,
   pinnedCoins: any,
-  hiddenCoins: any,
-  includePlaceholder = false,
-  isLoadingAssets: boolean
+  hiddenCoins: any
 ) => {
-  let standardAssets: any = [],
-    pinnedAssets: any = [],
-    smallAssets: any = [],
-    hiddenAssets: any = [];
-
-  const { addedEth, assets } = addEthPlaceholder(
-    sortedAssets,
-    includePlaceholder,
-    pinnedCoins,
-    nativeCurrency,
-    isLoadingAssets
-  );
-
-  if (!assets.length) {
+  if (!sortedAssets.length) {
```

Comment:
return early if no assets to avoid extra computation + addition of coin divider row on wallet screen